### DISCUSSION TITLE
No NAS support after 2014 - solution?

### DISCUSSION THREAD

**Discussion by jkolko** [AxisCommunications/discussions#244] [General]
Hi - Axis isn't going to support third party NAS recording after 2024. For those of you not interested in their proprietary NVRs - what are you going to do as a solution?

**Comment by vivekatoffice** [comment]
Hi @jkolko ,
What I understand from [Removed support for SMB 1.0 and 2.0](https://help.axis.com/en-us/axis-os#changes-in-axis-os-12-0) is that SMB 3.0 will be available:

![image](https://github.com/AxisCommunications/discussions/assets/52032125/72d5da5e-9250-43ac-bb40-2a603b1e6f97)
